src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -0,0 +1,32 @@
+import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
+import { CityWeatherShortcutLoader } from 'components/CityWeatherShortcut/components/CityWeatherShortcutLoader';
+import { Place } from 'models/Place';
+import { Container } from './NearestCitiesWeatherWidget.styled';
+import { NearestCitiesWeatherWidgetTestIds } from './NearestCitiesWeatherWidgetTestIds';
+
+interface Props {
+  loading: boolean;
+  cityName: string | undefined;
+}
+
+const NearestCitiesWeatherWidget = ({ loading, cityName }: Props) => {
+  const renderContent = () => {
+    if (loading) {
+      return [...Array(6)].map((_, id) => {
+        return <CityWeatherShortcutLoader key={id} />;
+      });
+    }
+
+    if (cityName) {
+      return ([] as Place[]).map((city) => {
+        return <CityWeatherShortcut key={city.id} cityName={city.tags.name} />;
+      });
+    }
```

Comment:
**suggestion** It could be shortened
```suggestion
    if (loading) {
      return [...Array(6)].map((_, id) => <CityWeatherShortcutLoader key={id} />);
    }

    if (cityName) {
      return ([] as Place[]).map((city) => <CityWeatherShortcut key={city.id} cityName={city.tags.name} />);
    }
```